src/h3/mod.rs

```diff
@@ -252,7 +252,7 @@ use std::collections::VecDeque;
 use crate::octets;
 
 /// The current HTTP/3 ALPN token.
```

Comment:
The comment is a bit weird with this change.

Reply:
Fixed. I also expanded this comment with more details on how to use the value.